### PR TITLE
swap installer signing to eclipse foundation

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

**Comment by gdams** [comment]
run tests

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -630,6 +625,62 @@ class Build {
         }
     }
 
+    def signInstaller(VersionInfo versionData) {
+        if (versionData == null || versionData.major == null) {
+            context.println "Failed to parse version number, possibly a nightly? Skipping installer steps"
+            return
+        }
+
+        context.node('master') {
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:634]: does it have to be master?

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -630,6 +625,62 @@ class Build {
...
+    private void signInstallerJob(VersionInfo versionData) {
+        def filter = ""
+
+        switch (buildConfig.TARGET_OS) {
+            case "mac": filter = "**/OpenJDK*_mac_*.pkg"; break
+            case "windows": filter = "**/OpenJDK*_windows_*.msi"; break
+            default: break
+        }
+
+        def nodeFilter = "eclipse-codesign"
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:660]: worth parameterizing?

**Review by karianna — CHANGES_REQUESTED** [review]

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 2 above)
Comment (gdams) [reply pipelines/build/common/openjdk_build_pipeline.groovy:660]: this is very specific code to us, it's unlikely that anyone else would consume this part of the pipeline

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (gdams) [reply pipelines/build/common/openjdk_build_pipeline.groovy:634]: yes

**Review by karianna — APPROVED** [review]
LGTM

**Review by andrew-m-leonard — APPROVED** [review]

**Pull request merged**